crates/bevy_asset/src/asset_changed.rs

```diff
@@ -0,0 +1,475 @@
+//! Filter query to limit iteration over [`Handle`]s which content was updated recently.
```

Comment:
This comment feels out of place. What iteration are we talking about here? Is that really the proper documentation for this module? I'd expect something like that instead:

```suggestion
//! Query filter for asset change detection.
//!
//! This module contains the change detection mechanism of assets used to filter changes in system queries.
//! It's similar to the `Added<T>` and `Changed<T>` query filter used for components.
```

Or something along those lines.